src/garage/envs/bullet/bullet_env.py

```diff
@@ -1,65 +1,69 @@
-            is_image (bool): True if observations contain pixel values,
-                false otherwise. Setting this to true converts a gym.Spaces.Box
-                obs space to an akro.Image and normalizes pixel values.
+            args: positional arguments
+            kwargs: keyword arguments
+
+        Returns:
+             garage.envs.bullet.BulletEnv
 
         """
-        if not env:
-            # 'RacecarZedBulletEnv-v0' environment enables rendering by
-            # default, while pybullet allows only one GUI connection at a time.
-            # Setting renders to False avoids potential error when multiple
-            # of these envs are tested at the same time.
-            if env_name == 'RacecarZedBulletEnv-v0':
-                env = gym.make(env_name, renders=False)
-            else:
-                env = gym.make(env_name)
-
-        # Needed for deserialization
-        self._env = env
-        self._env_name = env_name
-
-        super().__init__(env)
-        self.action_space = akro.from_gym(self.env.action_space)
-        self.observation_space = akro.from_gym(self.env.observation_space,
-                                               is_image=is_image)
-        self._spec = EnvSpec(action_space=self.action_space,
-                             observation_space=self.observation_space)
-
-    @property
-    def spec(self):
-        """Return the environment specification.
-
-        This property needs to exist, since it's defined as a property in
-        gym.Wrapper in a way that makes it difficult to overwrite.
+        return super(BulletEnv, cls).__new__(cls)
 
-        Returns:
-            garage.envs.env_spec.EnvSpec: The envionrment specification.
+    def __init__(self, env, is_image=False, max_episode_length=None):
+        """Returns a wrapper class for bullet-based gym.Env.
+
+        Args:
+            env (gym.wrappers.time_limit or str): A gym.TimeLimit
+                object wrapping a gym.Env created via gym.make(). Or a name
+                of the gym environment to be created.
+            is_image (bool): True if observations contain pixel values,
+                false otherwise. Setting this to true converts a gym.Spaces.Box
+                obs space to an akro.Image and normalizes pixel values.
+            max_episode_length (int): The maximum steps allowed for an episode.
 
         """
-        return self._spec
+        env_name = None
+        if isinstance(env, str):
+            env_name = env
+        elif isinstance(env, gym.Env):
+            env_name = env.spec.id
+
+        # 'RacecarZedBulletEnv-v0' environment enables rendering by
+        # default, while pybullet allows only one GUI connection at a time.
+        # Setting renders to False avoids potential error when multiple
+        # of these envs are tested at the same time.
+        if 'RacecarZedBulletEnv' in env_name:
+            env = gym.make(env_name, renders=False)
+
+        super().__init__(env=env,
```

Comment:
```suggestion
        super().__init__(env,
```
Always call positional arguments as positional.